lib/web_ui/dev/firefox_installer_test.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@TestOn('vm')
```

Comment:
If this is only meant to run on Linux, I think you can do 'vm && linux'.

See also: https://github.com/dart-lang/test/tree/master/pkgs/test#restricting-tests-to-certain-platforms

Reply:
Nice! Thank!